build.gradle.kts

```diff
@@ -168,6 +178,11 @@ testing.suites {
     targets.configureEach {
       testTask {
         maxParallelForks = Runtime.getRuntime().availableProcessors()
+        javaLauncher.set(
+          javaToolchains.launcherFor {
+            languageVersion.set(JavaLanguageVersion.of(testJdk))
+          }
```

Comment:
This defaults to `java.toolchain`, I guess you wanted to use `testJdk` in `testing` block.